package.json

```diff
@@ -71,9 +71,9 @@
         "@types/node": "^20.10.7",
         "@types/react": "18.2.43",
         "@types/react-dom": "18.2.17",
-        "@types/lodash.debounce": "^4.0.7",
```

Comment:
Removed this package from this [PR](https://github.com/deriv-com/quill-ui/pull/303/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) as it's not used anymore